ipyleaflet/leaflet.py

```diff
@@ -644,16 +644,16 @@ class SplitMapControl(Control):
     _view_name = Unicode('LeafletSplitMapControlView').tag(sync=True)
     _model_name = Unicode('LeafletSplitMapControlModel').tag(sync=True)
 
-    left_layer = Instance(TileLayer).tag(sync=True, **widget_serialization)
-    right_layer = Instance(TileLayer).tag(sync=True, **widget_serialization)
+    left_layer = Instance(Layer).tag(sync=True, **widget_serialization)
+    right_layer = Instance(Layer).tag(sync=True, **widget_serialization)
 
     @default('left_layer')
     def _default_left_layer(self):
-        return TileLayer()
+        return Layer()
```

Comment:
I guess the default value as a `TileLayer` was fine. It does not really make sense to create an instance of `Layer`.